fineract-provider/src/main/java/org/apache/fineract/infrastructure/creditbureau/domain/CreditBureauLoanProductMapping.java

```diff
@@ -24,26 +24,34 @@
 import javax.persistence.ManyToOne;
 import javax.persistence.OneToOne;
 import javax.persistence.Table;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.Accessors;
 import org.apache.fineract.infrastructure.core.api.JsonCommand;
 import org.apache.fineract.infrastructure.core.domain.AbstractPersistableCustom;
 import org.apache.fineract.portfolio.loanproduct.domain.LoanProduct;
 
 @SuppressWarnings({ "MemberName" })
 @Entity
 @Table(name = "m_creditbureau_loanproduct_mapping")
+@Getter
+@Setter
+@NoArgsConstructor
+@Accessors(chain = true)
 public class CreditBureauLoanProductMapping extends AbstractPersistableCustom {
 
-    @Column(name = "is_credit_check_mandatory")
-    private boolean isCreditCheckMandatory;
+    @Column(name = "credit_check_mandatory")
+    private boolean creditCheckMandatory;
 
     @Column(name = "skip_credit_check_in_failure")
     private boolean skipCreditCheckInFailure;
 
     @Column(name = "stale_period")
     private int stalePeriod;
 
-    @Column(name = "is_active")
-    private boolean isActive;
+    @Column(name = "active")
```

Comment:
Missing the liquibase script for this change.

Reply:
undone the change.